ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# PrestoDB: Mission and Architecture
+
+## Introduction
+
+PrestoDB is an open-source distributed SQL query engine designed for fast analytic queries against data sources of all sizes, ranging from gigabytes to petabytes. It was originally developed by Facebook and later made available to the broader community. It is governed by the Presto Foundation, a member of the Linux Foundation.
+
+## Mission
+
+### For users
+
+The primary mission of PrestoDB is to enable efficient and high-speed data processing for analytics and batch at scale. It aims to provide a single, unified query system that can access and process data stored in various formats and storage systems. Key aspects of its mission include:
+
+1. **High Performance:** Presto aims to be a single unified query engine suitable for low latency interactive workloads, adhoc and exploratory analytics, and large scale batch workloads. To achieve this, Presto works to achieve vertical integration over modular portions of the stack.
+2. **Scalability:** Presto is used to access large amounts of data from data lakes. As such, it aims for high degrees of scalability.
```

Comment:
Can we avoid the term "data lake"? Unless there's some way that Presto is processing unstructured data that I've missed.

Reply:
Same answer as here: https://github.com/prestodb/presto/pull/22356#discussion_r1546799934